src/View/Helper/FormHelper.php

```diff
@@ -2529,7 +2532,7 @@ protected function validateValueSources(array $sources): void
 
         if ($diff) {
             throw new InvalidArgumentException(sprintf(
-                'Invalid value source(s): %s. Valid values are: %s',
+                'Invalid value source(s): `%s`. Valid values are: `%s`.',
```

Comment:
Ideally each value should be individually wrapped in back ticks.

Reply:
Did the same on 2 other occurrences as well